[2025-04-08T23:13:11.488+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-08T23:13:11.510+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-08T23:12:51.000773+00:00 [queued]>
[2025-04-08T23:13:11.526+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-08T23:12:51.000773+00:00 [queued]>
[2025-04-08T23:13:11.528+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-08T23:13:11.706+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_raw_data> on 2025-04-08 23:12:51.000773+00:00
[2025-04-08T23:13:11.714+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=877) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-08T23:13:11.716+0000] {standard_task_runner.py:72} INFO - Started process 879 to run task
[2025-04-08T23:13:11.717+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'extract_raw_data', 'manual__2025-04-08T23:12:51.000773+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmp4_le25yf']
[2025-04-08T23:13:11.718+0000] {standard_task_runner.py:105} INFO - Job 30: Subtask extract_raw_data
[2025-04-08T23:13:11.939+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.extract_raw_data manual__2025-04-08T23:12:51.000773+00:00 [running]> on host 5a8aeda502d0
[2025-04-08T23:13:12.999+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-08T23:12:51.000773+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-08T23:12:51.000773+00:00'
[2025-04-08T23:13:13.000+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-08T23:13:13.014+0000] {driven_data_pipeline.py:147} INFO - Started batch processing for 2025-04-08.
[2025-04-08T23:13:13.014+0000] {driven_data_pipeline.py:34} INFO - Created synthetic data for RO country code.
[2025-04-08T23:13:14.123+0000] {driven_data_pipeline.py:104} INFO - Written 2101 records to the CSV file.
[2025-04-08T23:13:14.324+0000] {driven_data_pipeline.py:120} INFO - Added UUID to the dataset.
[2025-04-08T23:13:14.325+0000] {driven_data_pipeline.py:155} INFO - Finished batch processing 2025-04-08.
[2025-04-08T23:13:14.325+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-08T23:13:14.338+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-08T23:13:14.339+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=extract_raw_data, run_id=manual__2025-04-08T23:12:51.000773+00:00, execution_date=20250408T231251, start_date=20250408T231311, end_date=20250408T231314
[2025-04-08T23:13:14.636+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-08T23:13:14.659+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-08T23:13:14.748+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
